### ISSUE TITLE
Icon Request: fa-medicare

### ISSUE THREAD

**Issue by yesi79** [FortAwesome/Font-Awesome#9988]
Medicare icon to be used within a healthcare data app
![medicare](https://cloud.githubusercontent.com/assets/539719/18966421/14db0142-864f-11e6-8093-e9c7da5a6002.png)

**Comment by tagliala** [comment]
Is this a brand?

**Comment by tcalice** [comment]
Very useful idea to add the HHS logo to Font Awesome

**Comment by tagliala** [comment]
is this a brand? could you please provide more details about this icon?

**Comment by tcalice** [comment]
Health and Human Services (HHS) is a large (1 trillion dollar per year budget) department of the United States.

The HHS Brand ("the triple eagle") icon is a combination of three silhouettes that form the shape of an eagle.  The shape on the left is an eagle (the symbol of the country, the United States of America).  The shape in the center is a person (representing citizens) and the shape on the right is a smaller person (representing the vulnerable citizens).

**Comment by tcalice** [comment]
To tie this back to original post by yesi79... Medicare is an agency within the Department of Health and Human Services (HHS)

Bottom line: The HHS Department logo is reused by the agencies within the department (Medicare being one of them).

**Comment by yesi79** [comment]
Yes, this is a government brand: https://www.cms.gov/About-CMS/Agency-Information/CMS-Brand-Identity/

Very useful in health applications, was hoping we could have this icon similar to how Facebook, twitter and Instagram icons are available.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
